src/Functions/Functions/ChangeLog.md

```diff
@@ -18,7 +18,8 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
-* Upgraded Microsoft.Web API version to 2023-12-01 and added support for creating function apps on container apps
+* Upgraded to Microsoft.Web API version 2023-12-01 [#25347]
+* Added support for creating function apps on container [#22736]
```

Comment:
Please update the second bullet point to be: `Added support for creating function apps on container apps`. Thank you.